scripts/runTestsOnTravis.sh

```diff
@@ -35,7 +35,7 @@ elif [ "$1" = "js_tests" ]; then
     # tests by using the `skip_js_cluster_tests` and `skip_js_super_cluster_tests`
     # build tags.
 
-    go test -race -v -run=TestJetStream ./server -tags=skip_js_cluster_tests,skip_js_super_cluster_tests,skip_js_cluster_chaos_tests -count=1 -vet=off -timeout=30m -failfast
+    go test -race -v -run=TestJetStream ./server -tags=skip_js_cluster_tests,skip_js_super_cluster_tests,skip_js_chaos_tests -count=1 -vet=off -timeout=30m -failfast
```

Comment:
I really would want to use the reverse, that is, not have flags here but instead have a "positive" tag when you want to run the chaos tests (in a section down below).

Reply:
I'm working on the reversed tag. 
I just wanted to get this fix out in case someone is unable to pass PR build because chaos tests are failing

Reply:
They were not running for some reason... I don't see them in the run on main for instance..

Reply:
Probably the other build tags were excluding them

```
//go:build !skip_js_tests && !skip_js_cluster_tests && !skip_js_chaos_tests
```

Reply:
I think this is because you already had `!skip_js_cluster_tests` in the chaos tests?